agents/tests/tests.py

```diff
@@ -122,7 +122,10 @@ def run(self, result=None):
     @classmethod
     def tearDownClass(cls):
         agent.terminate()
+        # workaround: child process not killed by agent terminate/kill
+        os.system("pkill -f aea")
         os.system("pkill -f libp2p_node")
```

Comment:
So since the `agent` process runs in shell, the actual shell is being terminated here and the child processes keep running. The current approach is brute force. We need to fix it eventually.

Reply:
https://docs.python.org/2/library/multiprocessing.html#multiprocessing